rhino3dm/py/SampleCreateCircleAndWriteTo3dmFile.py

```diff
@@ -0,0 +1,11 @@
+import rhino3dm
+import random
+
+model = rhino3dm.File3dm()
+for i in range(20):
+	pt = rhino3dm.Point3d(random.uniform(-10,10), random.uniform(-10,10), 0)
+	model.Objects.AddPoint(pt)
+	circle = rhino3dm.Circle(pt, random.uniform(1,4))
+	model.Objects.AddCircle(circle)
+	
+model.Write("circle.3dm", 6)
```

Comment:
Please remove the 6 from this call. Our samples should be writing the current Rhino version.

Reply:
Ok, corrected rhino version